h5p_proplist.go

```diff
@@ -9,16 +9,20 @@ package hdf5
 // #include <string.h>
 // inline static
 // hid_t _go_hdf5_H5P_DEFAULT() { return H5P_DEFAULT; }
+// hid_t _go_hdf5_H5P_DATASET_CREATE() { return H5P_DATASET_CREATE; }
```

Comment:
perhaps make it `inline static` as the one above?

Reply:
Fixed